### DISCUSSION TITLE
Pids_limit

### DISCUSSION THREAD

**Discussion by itmwiw** [cri-o/cri-o#6473] [Q&A] [answered]
Hello,
First of all, great work everyone!
I have a question regarding Pids_limit. In fact I'd like to know if it's a processes or threads limit. And if It's a processes limit, is there any limit for the threads as well.
Thanks in advance.
Regards,
Tarik

**Comment by haircommander** [comment, marked as answer]
it's both thread and process. the `pids_limit` parameter directly maps to the pids limit in the kernel, which uses it to enforce both processes and threads. I don't have a direct source on that, but [this lwn article](https://lwn.net/Articles/784831/) mentions it, as does [this cri-o](https://github.com/cri-o/cri-o/issues/1921) issue

**Reply by itmwiw** [reply]
thanks @haircommander for the answer.
